qiita_pet/templates/sitebase.html

```diff
@@ -133,6 +133,14 @@
                 <li><a href="/study/list/">View Studies</a></li>
               </ul>
             </li>
+            <li class="dropdown">
+              <a href="/user/messages/">Messages</a>
+              <ul class="dropdown-menu">
+              {% for msg, timestamp in current_user.messages(5) %}
```

Comment:
because this is on sitebase, so without this doing it this way we need every single template call to pass messages, which is inefficient.